configuration/src/test/java/io/airlift/configuration/TestConfig.java

```diff
@@ -204,6 +205,33 @@ public void testDetectsNoConfigAnnotations()
         }
     }
 
+    @Test
+    public void testDetectsSecuritySensitiveConfig()
+    {
+        try {
+            Injector injector = createInjector(Collections.<String, String>emptyMap(), ImmutableList.of("access-key"), createModule(Key.get(SecuritySensitiveConfig.class), SecuritySensitiveConfig.class, null));
```

Comment:
I understand that this provides a form of a unit test that the annotation is there.
However, that means i either need to remember to add annotation or to add a test (and then add annotation), which relies on remembering.

so the add value of this is that, once i have a test, one cannot remove the annotation without updating the test